csharp/extractor/Semmle.Extraction.CSharp/Entities/Types/Type.cs

```diff
@@ -285,10 +285,10 @@ public void PopulateGenerics()
                     Context.CreateEntity(member);
                 }
 
-                if (Symbol.BaseType is not null)
+                if (Symbol.BaseType is not null && Symbol.BaseType.ShouldExtractSymbol())
```

Comment:
Is this change required? The base type cannot be less accessible than the currently processed type: https://learn.microsoft.com/en-us/dotnet/csharp/misc/cs0060. Is there any case that we're extracting a type without its base type?

Reply:
You are right - this is not required.